src/pages/index.js

```diff
@@ -2,32 +2,36 @@ import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import ModalWithImage from "../components/ModalWithImage.js";
 import ModalWithForm from "../components/ModalWithForm.js";
+import ModalConfirmation from "../components/ModalConfirmation.js";
 import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
 import {
+  selectors,
   //cards
-  initialCards,
-  cardList,
-  cardData,
+  /*   cardList,
+  cardData, */
   cardSelector,
   //profile
   profileEditForm,
   nameInput,
```

Comment:
Needs correction: The profile info can't be updated on the server

Reply:
The info is now updated on the server